src/test/java/com/feedzai/commons/sql/abstraction/engine/impl/abs/EngineGeneralTest.java

```diff
@@ -168,7 +168,9 @@
 
     @Parameterized.Parameters
     public static Collection<DatabaseConfiguration> data() throws Exception {
-        return DatabaseTestUtil.loadConfigurations();
+        return DatabaseTestUtil.loadConfigurations(
+                "h2Legacy", "h2RemoteLegacy", "mysql", "sqlserver", "postgresql", "cockroach", "oracle", "db2"
```

Comment:
Why this change?

Reply:
Because this test class needs do load configurations of all of the database types, except non-legacy H2 (the specific non-legacy H2 tests are in another class - `H2NotLegacyTest`). So it is necessary to specify each one explicitly.